docs/api/classifier/rocauc.rst

```diff
@@ -18,29 +21,74 @@ sensitivity and specificity.
     classes = ["unoccupied", "occupied"]
 
     # Extract the numpy arrays from the data frame
-    X = data[features].as_matrix()
-    y = data.occupancy.as_matrix()
+    X = data[features].values
+    y = data.occupancy.values
 
     # Create the train and test data
     X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)
 
 .. code:: python
 
+    from yellowbrick.classifier import ROCAUC
     from sklearn.linear_model import LogisticRegression
 
-    from yellowbrick.classifier import ROCAUC 
+    # Instantiate the visualizer with the classification model
+    visualizer = ROCAUC(LogisticRegression(), classes=classes)
+
+    visualizer.fit(X_train, y_train)  # Fit the training data to the visualizer
+    visualizer.score(X_test, y_test)  # Evaluate the model on the test data
+    g = visualizer.poof()             # Draw/show/poof the data
+
+
+.. image:: images/rocauc_binary.png
+
+
+.. warning::
+    Binary classification using a Scikit-learn-style estimator with only a
+    ``decision_function``, triggers an ``IndexError`` because the predictions
+    will be a 1D array, meaning there is only sufficient information to plot a
+    single curve. More on this bug can be found in this `notebook <https://github.com/DistrictDataLabs/yellowbrick/blob/develop/examples/rebeccabilbro/rocauc_bug_research.ipynb>`_. The bug was addressed in a `July 2018 PR <https://github.com/DistrictDataLabs/yellowbrick/pull/533>`_
+    and will be fixed in v0.9, where the solution will be to set the ``micro``,
+    ``macro``, and ``per-class`` parameters of ``ROCAUC`` to ``False``.
+
+
+Multi-class ROCAUC Curves
+#########################
+
+Yellowbrick's ``ROCAUC`` Visualizer does allow for plotting multiclass classification curves.
+ROC curves are typically used in binary classification, and in fact the Scikit-Learn ``roc_curve`` metric is only able to perform metrics for binary classifiers. Yellowbrick addresses this by binarizing the output (per-class) or to use one-vs-rest (micro score) or one-vs-all (macro score) strategies of classification.
+
+.. code::
+
+    # Load multi-class classification dataset
+    data = load_game("occupancy")
+
+    classes = ["win", "loss", "draw"]
+
+    # Encode the non-numeric columns
+    game.replace({'loss':-1, 'draw':0, 'win':1, 'x':2, 'o':3, 'b':4}, inplace=True)
```

Comment:
Where is the ``game`` variable coming from?

Reply:
Here `game` is not a variable but an dataframe that holds the win, lose or draw game dataset - hopefully that is clarified in my most recent push!

Reply:
When loading the new datasets, I believe that you don't have to pass "game" to the function. You should be able to simply use: `game = load_game()`

Reply:
Thanks @Kautumn06! Just updated that.

Reply:
Exactly @Kautumn06 that is what I was trying to say

Reply:
Ok gotcha @lwgray ! Fixed now.